### DISCUSSION TITLE
Using TLMExplorer to replace lost VIRB Edit functionality

### DISCUSSION THREAD

**Discussion by davefolts** [M-Yankov/TelemetryExporter#36] [General]
Good Afternoon-
This is an excellent utility!
It seems that a few years ago Garmin stopped supporting the "**Aviation**" capability of the VIRB camera and for that matter specifically, the Roll, Pitch and Yaw data in the .fit file as well as Vx and Vy  (used to calculate the rate of climb, descent and drift)
All of those parameters are useful in producing education video for flight training, especially when attempting to display the "**Artificial Horizon**" or HUD widget that is still in the VIRB Edit library, but sadly no longer driven by VIRB edit processing of roll, pitch, yaw, Vx no Vy data in the .fit file.

Could you prioritize your next work on this utility by adding roll, pitch and yaw processing please?

thank you.

r
Dave Folts

**Comment by M-Yankov** [comment]
@davefolts Can you check the comments in #37 ?
